infra/Pos/Infra/Diffusion/Types.hs

```diff
@@ -26,6 +30,13 @@ import           Pos.Core.Chrono (OldestFirst (..))
 
 
 
+data StreamEntry = StreamEnd | StreamBlock !Block
+
+data DiffusionHealth = DiffusionHealth {
```

Comment:
This type needs documentation. What do the gauges mean? Are they relevant to serving the stream, or downloading?

Reply:
Will document. The counters  can be used to see if the current block syncing performance is limited by the network or by local CPU/disk io.